java/src/jmri/jmris/AbstractReporterServer.java

```diff
@@ -106,12 +103,10 @@ class ReporterListener implements PropertyChangeListener {
         public void propertyChange(PropertyChangeEvent e) {
             // If the Commanded State changes, show transition state as "<inconsistent>"
             if (e.getPropertyName().equals("currentReport")) {
-                String now;
-                try {
-                    now = e.getNewValue().toString();
-                } catch (NullPointerException npe) {
-                    // current report is null, which is expected.
-                    now = null;
+                var n = e.getNewValue();
+                String now = null; // current report is null, which is expected.
```

Comment:
This comment is wrong. It made sense before when you had a try catch checking for NPE. After the change, it would be better if the comment is:
`// current report might be null`